Adafruit_ILI9341.h

```diff
@@ -155,6 +156,19 @@ class Adafruit_ILI9341 : public Adafruit_GFX {
         void      drawFastHLine(int16_t x, int16_t y, int16_t w, uint16_t color);
         void      fillRect(int16_t x, int16_t y, int16_t w, int16_t h, uint16_t color);
 
+	// Compat with ILI9341_T3 lib
```

Comment:
we need to figure out if the names for the methods make sense with the GFX library, or if we need to move them there...